README.md

```diff
@@ -2,57 +2,66 @@
 
 **This module currently only supports deploying a single task per service.**
 
-Terraform module which creates a ECS service to be deployed on the specified cluster.  
+Terraform module which creates a ECS service to be deployed Fargate or EC2.
 
 ## Examples
 
 ### Internal Service
 ```
-data "aws_iam_role" "task" {
-  name = "${var.service_name}_task"
-}
-
 module "ecs_service" {
   source                   = "git::https://github.com/bnc-projects/terraform-ecs-service.git?ref=1.0.0"
   application_path         = "/v1/service" 
   cluster_name             = "ecs-cluster-name"
-  docker_image             = "bncprojects/${var.service_name}:${var.service_version}"
   external_lb_listener_arn = ""
+  external_lb_name         = ""
   internal_lb_listener_arn = "arn:aws:elasticloadbalancing:region:account-id:listener/app/intlb"
-  java_options             = "-javaagent:newrelic/newrelic.jar -Dnewrelic.environment=${terraform.workspace} -Dnewrelic.config.file=newrelic/newrelic.yml"
+  internal_lb_name         = "intlb"
   is_exposed_externally    = false
   priority                 = 1
-  service_name             = "${var.service_name}
-  splunk_token             = "${var.splunk_token}
-  splunk_url               = "${var.splunk_url}
-  spring_profile           = "default,${terraform.workspace}"
-  task_role_arn            = "${aws_iam_role.task.arn}"
+  service_name             = var.service_name
+  task_definition_arn      = aws_ecs_task_definition.bar.arn
   vpc_id                   = "vpc-124552462"
-  tags                     = "${merge(local.common_tags, var.tags)}"
+  tags                     = merge(local.common_tags, var.tags)}
```

Comment:
I notice that we have a mixture of syntax for `tags`, typically we see 

```
"${merge(local.common_tags, var.tags)}"
```

Is the syntax here correct? There also appears to be trailing `}` on the edited line.

Reply:
Both are valid for us as we use HCL2. However we should be using the HCL2 way for consistency. Our  modules are now only compatible with Terraform 0.12.6 or higher.